lib/miq_expression.rb

```diff
@@ -2,6 +2,8 @@ class MiqExpression
   require_nested :Tag
   include Vmdb::Logging
   attr_accessor :exp, :context_type, :preprocess_options
+  ALLOWED_OPERATORS_EXPRESSION_AS_VALUE = %w(= > >= < <= !=)
```

Comment:
Would prefer to avoid these. But I can appreciate that this may be necessary.
@alongoldboim will comment what tests this fixes